internal/node/node_loader.js

```diff
@@ -433,15 +432,15 @@ module.constructor._resolveFilename = function(request, parent) {
   // If import was not resolved above then attempt to resolve
   // within the node_modules filegroup in use
   try {
-    const resolved = originalResolveFilename(
-        resolveRunfiles(undefined, NODE_MODULES_ROOT, request), parent);
+    const resolved = originalResolveFilename(resolveRunfiles(undefined, NODE_MODULES_ROOT, request), parent, isMain, options);
     if (DEBUG)
       console.error(
           `node_loader: resolved ${request} within node_modules (${NODE_MODULES_ROOT}) to ` +
           `${resolved} from ${parentFilename}`
       );
     return resolved;
   } catch (e) {
+    console.error(e);
```

Comment:
do you mean to leave this here? seems like it could be very spammy even for a successful execution where some other resolution succeeds

Reply:
Whoops sorry. That was left from debugging. Will remove.